api/proxy/proxy.go

```diff
@@ -176,26 +176,34 @@ func (p *proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		r.SetBasicAuth(username, password)
 	}
 
-	// Forward old ocis domain always to new url
-	if p.isOcisRequest(r) {
+	// TEMPORARY REDIRECT
+	// People may have bookmarked old URLs, we provide a redirection for those
+	// for example, old.cernbox.cern.ch/index.php/s/... -> cernbox.cern.ch/index.php/s/...
+	// The translation of the urls is done by the web frontend
+	if p.isOldWebURLToMigrateToOCIS(r) {
```

Comment:
It's an "ocis" domain (`new.cernbox.cern.ch`) and not old. Calling it "oldweb" is confusing, since we already have an old web (php).